### PR TITLE
Environmental sensor OIC resource

### PR DESCRIPTION
**Pull request by kiendinh**
It uses noble to connect via BLE to the environmental sensor to get data.
There are temperature, humidity, air pressure and UV index sensor data that
the resource represents. To use, launch the nodejs application in the
homegateway with BLE capability. It has been tested on MinnowBoard Turbot &
Galileo with a CSR 4.0 BLE USB dongle. Edison is not yet supported.

It also support simulation mode for testing.

### CODE REVIEW

**Comment by gvancuts** [comment]
👍

File: ocf-servers/doc/environment.raml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        if:
+          enum: ["oic.if.s"]
+
+/EnvironmentSensorResURI:
+  description: |
+    This resource describes an environmental sensor
+    temperature (Celcius), humidity (percentage) and pressure (hPa) are float numbers,
+    while uvIndex is an integer. They represent the sensed environemental data.
+
+  displayName: Environtmental Sensor
```
Comment (gvancuts) [inline ocf-servers/doc/environment.raml]: s/Environtmental/Environmental

File: ocf-servers/doc/environment.raml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.s"]
+
+/EnvironmentSensorResURI:
+  description: |
+    This resource describes an environmental sensor
+    temperature (Celcius), humidity (percentage) and pressure (hPa) are float numbers,
+    while uvIndex is an integer. They represent the sensed environemental data.
```
Comment (gvancuts) [inline ocf-servers/doc/environment.raml]: s/environemental/environmental

**Comment by kiendinh** [comment]
Thank you for checking it out. I just pushed the modifications.

**Pull request merged**